WordPress/src/test/java/org/wordpress/android/ui/sitecreation/domains/SiteCreationDomainValidatorTest.kt

```diff
@@ -0,0 +1,27 @@
+package org.wordpress.android.ui.sitecreation.domains
+
+import junit.framework.Assert.assertFalse
+import junit.framework.Assert.assertTrue
+import org.junit.Test
+import org.wordpress.android.RobolectricSetupTest
+import org.wordpress.android.util.UrlUtilsWrapper
+
+class SiteCreationDomainValidatorTest : RobolectricSetupTest() {
```

Comment:
Also for the tests, I utilized `Robolectric` so that I could get a working version of the `UrlUtilsWrapper` to power the domain validation logic. What do you think?

Reply:
Since the implementation has been simplified I can now remove Robolectric.

Reply:
Removed this in e9e7275